src/utils/url.js

```diff
@@ -1,12 +1,18 @@
-import { environment } from "../environments/environment";
-
 const URL = {
     login: () => {
         return `login`;
     },
 
+    signup: () => {
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/abdulqhayum47/lets-react/pullRequest?prid=9741124)
```suggestion
  signup: () => {
```